### PR TITLE
Feature/handle-media

### PR DESCRIPTION
**Pull request by debreb**
Open imageLibrary,
 add image,
delete image

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -20,18 +20,33 @@
       "infoPlist": {
         "NSLocationAlwaysAndWhenInUseUsageDescription": "App requires geolocation to retrieve the current location in order to be easier to mark the hotspots.",
         "NSLocationAlwaysUsageDescription": "App requires geolocation to retrieve the current location in order to be easier to mark the hotspots.",
-        "NSLocationWhenInUseUsageDescription": "App requires geolocation to retrieve the current location in order to be easier to mark the hotspots."
+        "NSLocationWhenInUseUsageDescription": "App requires geolocation to retrieve the current location in order to be easier to mark the hotspots.",
+        "NSCameraUsageDescription": "App requires access to your phone’s camera.",
```
Comment (mveres) [inline app.json:24]: we need more descriptive messages, Apple won't accept generic messages on app review...
"Camera access is required to take photos of cats"  or something similar.

File: app.json [hunk 2]
```diff
@@ -20,18 +20,33 @@
...
+        "NSPhotoLibraryAddUsageDescription": "This app requires access to the photo library."
```
Comment (mveres) [inline app.json:25]: same here

**Review by mveres — CHANGES_REQUESTED** [review]
for the image-placeholder and other assets in general we need a resolution at least 2 or 3x better than the size displayed for pixel perfect rendering
e.g. if we need to render an icon 64px x 64px, then the asset should be 128px x 128px or 256x256.

File: components/CatCard.tsx [hunk 3]
```diff
@@ -230,19 +229,24 @@ export const CatCard = ({
...
-                data={[] /*cat.media*/}
-                keyExtractor={(item, index) => index.toString()}
-                renderItem={({ item }) => (
-                  <View>
-                    <Image style={styles.media} source={{ uri: item }} />
-                  </View>
-                )}
-              />
-            )}
+            {/* {!isEmpty(cat.media) && ( */}
```
Comment (mveres) [inline components/CatCard.tsx:232]: please remove unused code

File: components/CatCard.tsx [hunk 4]
```diff
@@ -230,19 +229,24 @@ export const CatCard = ({
...
+                ] /*cat.media*/
+              }
+              keyExtractor={(item, index) => index.toString()}
+              renderItem={({ item }) => (
+                <View>
+                  <Image style={styles.media} source={{ uri: item }} />
+                </View>
+              )}
+            />
+            {/* )} */}
```
Comment (mveres) [inline components/CatCard.tsx:249]: same

File: components/AddCatCard.tsx [hunk 5]
```diff
@@ -464,11 +522,30 @@ export const AddCatCard = ({
         {!isEditingMode && (
           <Caption style={styles.textInputTitle}>Adaugă poze/video</Caption>
         )}
-        <View style={{ paddingTop: 8 }}>
-          <Image style={styles.media} source={imagePlaceholder} />
+        <View style={styles.imagesView}>
+          <View style={{ maxWidth: '80%' }}>
```
Comment (mveres) [inline components/AddCatCard.tsx:526]: let's not have inline styles

File: components/CatCard.tsx (hunk 3 above)
Comment (debreb) [reply components/CatCard.tsx:232]: at this moment is unused, but we need it when we are working with the real data

**Pull request merged**